### PR TITLE
Add `:help auto` + link to it from relevant errors and docs

### PR DESCRIPTION
**Pull request by oskarhane**
A follow-up PR for https://github.com/neo4j/neo4j-browser/pull/1048.

* Add new `auto` help page
* Add link to help page from `USING PERIODIC COMMIT` Cypher error frame
* Add links to auto help page from other relevant help pages

Screenshot 2020-01-30 10 49 56
Screenshot 2020-01-30 10 50 01
Screenshot 2020-01-30 10 50 07
Screenshot 2020-01-30 10 49 50

### CODE REVIEW

File: src/browser/documentation/help/auto.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ */
+
+import React from 'react'
+const title = 'Auto-committing transactions'
+const subtitle = 'Execute a Cypher query within an auto-committing transaction'
+const category = 'browserUiCommands'
+const content = (
+  <>
+    <p>
+      The <code>:auto</code> command will send the Cypher query after is in an
```
Comment (HerrEmil) [inline src/browser/documentation/help/auto.jsx]: `after is in` should probably be `after it in`

File: src/browser/documentation/help/auto.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      auto committing transaction. In general this is not recommended because of
+      the lack of support for auto retrying on leader switch errors in clusters.
+      <br />
+      Some query types does however need to be sent in auto-committing
```
Comment (HerrEmil) [inline src/browser/documentation/help/auto.jsx]: Small grammar change: `Some query types does` -> `Some query types do`

File: src/browser/documentation/help/auto.jsx (hunk 1 above)
Comment (oskarhane) [reply src/browser/documentation/help/auto.jsx]: Thanks, updated

File: src/browser/documentation/help/auto.jsx (hunk 2 above)
Comment (oskarhane) [reply src/browser/documentation/help/auto.jsx]: Thanks, updated

**Comment by oskarhane** [comment]
Thanks for the review @HerrEmil! I've addressed your comments now.

**Pull request merged**